### PR TITLE
hw1 Yucheng

### PR DESCRIPTION
**Pull request by Yucheng98**
I used headers, bullet points, bold and italic features, as well as imbedded a website link and a picture. Everything went well except for the image. I now know that we need to pull from Github every time when uploading files onto my repository, and we cannot name the files with spaces.

### CODE REVIEW

**Comment by rsasschicago** [comment]
Hi Yucheng. Great job on the homework! You utilized several R Markdown functions beautifully.  I especially liked seeing your art! I'll pass on a piece of input that I received on my own homework, which is that it might be good for the photo to be resized. Other than that, I think you're execution is perfect.

Sincerely,

Rachel

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+During spare time, I like to
+
+* Keep succulent plants
+* Watch Japanese animation
+* Carve erasers into stamps
+
+Here is a picture of one of the eraser stamps I carved.
+
+![Image of Eraser carving](EraserCarving.jpg)
```
Comment (stanxie28) [inline README.md:21]: That is really cool! I really like that! Maybe you can slightly shrink the photo next time 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+I am currently still living in China due to the pandemic, so I am eager to meet everyone in person. Hope to see you soon when everything goes back to normal!
```
Comment (stanxie28) [inline README.md:23]: Great presentation! You used nearly every point that this homework requires - you can try to include a hyperlink next time!

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
 # hw01
+
+# Self Introduction
+
+### Background
+
+My name is **Yucheng Lu**, and I am originally from Shanghai, China.  I just finished my undergraduate degree in *Wake Forest University* with a major in Psychology and a minor in Japanese.  I thought the mascot of our school is funny, so here is a link to his picture.
```
Comment (ktownsend3) [inline README.md:7]: Hi, Yucheng! Your bio was very interesting, and I really enjoyed learning about you. You started off strong and already included 3 of the 5 elements in your first section; great job!

File: README.md (hunk 1 above)
Comment (ktownsend3) [inline README.md:21]: Great job including a photo! This is something I struggled with myself, so it is helpful to see it done correctly two different ways in your project.

File: README.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+During spare time, I like to
+
+* Keep succulent plants
+* Watch Japanese animation
+* Carve erasers into stamps
+
+Here is a picture of one of the eraser stamps I carved.
+
+![Image of Eraser carving](EraserCarving.jpg)
+
```
Comment (ktownsend3) [inline README.md:22]: You made great use of Markdown's features. You definitely seem comfortable with it and your introduction looks great overall!

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Reproducibility**         |           |         x     |            |

**Pull request closed without merging**